ci-docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+ARG TARGET_DOCKER_PLATFORM
+
+FROM library/debian:stretch
+
+RUN apt-get update && apt-get install -y qemu-user-static
+
+FROM $TARGET_DOCKER_PLATFORM/debian:stretch
+
+ARG HOST_DEBIAN_PLATFORM
+
+COPY --from=0 /usr/bin/qemu-* /usr/bin/
+
+RUN dpkg --add-architecture $HOST_DEBIAN_PLATFORM
+
+RUN apt-get update && apt-get install -y openjdk-8-jdk-headless:$HOST_DEBIAN_PLATFORM
+
+ENV PATH "/usr/lib/jvm/java-8-openjdk-$HOST_DEBIAN_PLATFORM/bin:$PATH"
+
+ENV SBT_LAUNCHER_VERSION 1.2.0
+
+RUN apt-get install -y wget curl
+
+# Install sbt
+RUN \
+  curl -L -o sbt-$SBT_LAUNCHER_VERSION.deb https://dl.bintray.com/sbt/debian/sbt-$SBT_LAUNCHER_VERSION.deb && \
+  dpkg -i sbt-$SBT_LAUNCHER_VERSION.deb && \
+  rm sbt-$SBT_LAUNCHER_VERSION.deb && \
+  apt-get update && \
+  apt-get install -y sbt
+
+RUN echo "deb http://deb.debian.org/debian stretch-backports main contrib non-free" >> /etc/apt/sources.list
+
+RUN apt-get update && apt-get install -y clang-5.0 zlib1g-dev libgc-dev libre2-dev
+
+ENV LD_LIBRARY_PATH "/usr/local/lib"
+
+ENV LC_ALL "C.UTF-8"
+
+ENV LANG "C.UTF-8"
+
+RUN useradd -ms /bin/bash scala-native
+
+USER scala-native
```

Comment:
Will this ensure that the owner of files created inside Docker match the Travis CI user?

Reply:
Some file unit tests check read/write permissions, which fail if the user is not set because the root user can do whatever it wants.